pkg/controller/cheinstallation/cheinstallation_controller.go

```diff
@@ -109,14 +117,37 @@ func (r *ReconcileCheInstallation) Reconcile(request reconcile.Request) (reconci
 	reqLogger.Info("Reconciling CheInstallation")
 
 	cheInstallation := &v1alpha1.CheInstallation{}
-	err := r.client.Get(context.TODO(), request.NamespacedName, cheInstallation)
+	err := r.client.Get(context.TODO(), types.NamespacedName{
+		Name: InstallationName,
+	}, cheInstallation)
```

Comment:
May I know why did you change it to the fixed name instead of using the one from the request?

Reply:
because when the request comes from the CheCluster resource which is watched, the `request.Namespace` argument to get the `CheInstallation` resource is incorrect

Reply:
gotcha - thanks for explanation :+1:

Reply:
my pleasure ;)